juju/utils.py

```diff
@@ -49,10 +49,12 @@ def juju_config_dir():
     config_dir = Path('~/.local/share/juju')
 
     # Check $JUJU_DATA
-    if juju_data := os.environ.get('JUJU_DATA'):
-        config_dir = Path(juju_data)
+    juju_data = os.environ.get('JUJU_DATA')
     # Secondly check: $XDG_DATA_HOME for ~/.local/share
-    elif xdg_data_home := os.environ.get('XDG_DATA_HOME'):
+    xdg_data_home = os.environ.get('XDG_DATA_HOME')
```

Comment:
No need to get this if the `juju_data` is available. Please put it in the else case down below.

Reply:
ack. Addressed by https://github.com/juju/python-libjuju/pull/993/commits/f05162e3d361d36b297afa3900e081bef1774b95